homeassistant/components/flexit/climate.py

```diff
@@ -14,12 +14,11 @@
     HVACAction,
     HVACMode,
 )
-from homeassistant.components.modbus import (
+from homeassistant.components.modbus import ModbusHub, get_hub
+from homeassistant.components.modbus.const import (  # pylint: disable=hass-component-root-import
```

Comment:
I disagree with this change. Things from a component that can be used outside of that component by another component should be imported from the main file to abstract the internals of the integration away as the depending integration should not care about where a constant is located.

This pylint check has been made for specifically this. Thus we should not disable it here.

Reply:
I happen to agree with you, but flexit uses a non-published, non-public interface, that is the primary problem.

Modbus do not have a supported API, so the choice was to fix the problem in flexit fast.

I have added another commit, that makes the constants local (interesting enough flexit already had a local version of the WRITE_REGISTER. This is not a good solution, but at least it allows modbus to continue on its path.